pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-central.md

```diff
@@ -5,138 +5,174 @@ title: Centreon Central
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Overview
+## Pack assets
 
-The Centreon Central Monitoring Connector will help you set up monitoring for your Centreon Central server. 
+### Templates
```

Comment:
```suggestion

The Centreon Central Monitoring Connector will help you set up monitoring for your Centreon Central server. 

> The best practice is to have the central server monitored by a poller if you have one. If not, you will need to add the `--hostname=''` option to the host's `EXTRAOPTIONS` macro to avoid host key verification issues.

### Templates
```